app/views/teachers/_form.html.erb

```diff
@@ -66,9 +66,9 @@
 
   <div class='form-group row'>
     <div class='col-12'>
-      <%= f.label :more_info, "More Information" %>
-      <%= f.text_field :more_info, placeholder: "ex. I'm teaching a college course",
-        class: "form-control" %>
+      <%= f.label :more_info, "More Information", class: "label-required" %>
+      <%= f.text_area :more_info, placeholder: "Please tell us why you need access.",
```

Comment:
Instead of a placeholder, can we put this below the form, like in this example?
https://getbootstrap.com/docs/4.0/components/forms/
And then actually, the `<strong>...` text I reference below could go inside the `<small>` tag.